src/jshint.js

```diff
@@ -4225,7 +4225,7 @@ var JSHINT = (function() {
     var i = 0;
     var inof = ["in", "of"];
     var level = 0; // BindingPattern "level" --- level 0 === no BindingPattern
-    var comma; // First comma punctuator at level 0
+    var firstcomma; // First comma punctuator at level 0
```

Comment:
The mass renaming was based off of @jugglinmike's feedback, which is valid because it looked like there was a few other places where global functions will be overwritten and cause jshint to not work